server.js

```diff
@@ -0,0 +1,87 @@
+//
+// This server is used only in docker images!
+// Dev server config is located in packages/blockchain-wallet-v4-frontend/webpack.config.js
+//
+const express = require('express')
+const compression = require('compression')
+const path = require('path')
+
+// store env configs
+const port = process.env.PORT || 8080
+const rootURL = process.env.ROOT_URL
+const webSocketURL = process.env.WEB_SOCKET_URL
+const apiDomain = process.env.API_DOMAIN
+const iSignThisDomain = process.env.I_SIGN_THIS_DOMAIN
+const isProduction = rootURL === 'https://blockchain.info'
+
```

Comment:
What do you think of adding a process.env.ENV=production instead, so we don't have to check against .info?

Reply:
👍